package.json

```diff
@@ -1,44 +1,39 @@
 {
-  "name": "react-native-heap-analytics",
-  "version": "3.0.0",
-  "description": "React Native wrapper for Heap Analytics.",
+  "name": "@heap/react-native-heap",
+  "version": "0.1.0-beta.1",
+  "description": "React Native event tracking with Heap.",
   "license": "MIT",
-  "author": "Andy Isaacson <andyi@heapanalytics.com>",
-  "homepage": "https://github.com/heap/react-native-heap-analytics",
-  "repository": "heap/react-native-heap-analytics",
+  "author": "Heap <http://www.heapanalytics.com>",
+  "homepage": "https://github.com/heap/react-native-heap",
+  "repository": "heap/react-native-heap",
   "keywords": [
     "analytics",
     "heap",
     "ios",
-    "jolt",
+    "android",
     "react-native"
   ],
   "main": "js/Heap",
   "files": [
     "android",
     "ios",
     "js",
-    "react-native-heap-analytics.podspec"
+    "react-native-heap.podspec"
   ],
+  "publishConfig": {
+    "access": "public"
```

Comment:
Where do we use this?

Reply:
Since we're using a "scoped" npm package (i.e. "@heap/"), the default for the publish action is for it to be private to our org, unless you issue the command with a special argument.  This just changes the default to be public despite it being scoped.